src/Microsoft.AspNetCore.Mvc.ViewFeatures/CookieTempDataProviderOptions.cs

```diff
@@ -22,5 +22,10 @@ public class CookieTempDataProviderOptions
         /// The domain set on a cookie. Defaults to <c>null</c>.
         /// </summary>
         public string Domain { get; set; }
+
+        /// <summary>
+        /// The name of the cookie which stores TempData. Defaults to <see cref="CookieTempDataProvider.CookieName"/>. 
```

Comment:
It might be worth renaming this to `CookieTempDataProvider.DefaultCookieName` unless it's a breaking change